Formula/q/quazip.rb

```diff
@@ -34,6 +34,9 @@ def install
   end
 
   test do
+    # Fails in Linux CI with "fatal error: zlib.h: No such file or directory"
```

Comment:
Have you tried adding
```ruby
uses_from_macos "bzip2"
uses_from_macos "zlib"
```
? (Though mostly the latter)

Reply:
We delete `CPATH` in test and `zlib` would be non-keg-only so I don't expect it to do much.

Maybe some other environment variables or updating `INCLUDEPATH`/`LIBPATH` could help.

Not sure if we should be embedding `HOMEBREW_PREFIX/include` as a default path of `qt`.

---

I mentioned system `zlib` since `/usr/include` is part of default paths in QMake, e.g.
`share/qt/mkspecs/features/toolchain.prf`
```
    unix:!darwin:if(!cross_compile|host_build) {
        isEmpty(QMAKE_DEFAULT_INCDIRS): QMAKE_DEFAULT_INCDIRS = /usr/include /usr/local/include
        isEmpty(QMAKE_DEFAULT_LIBDIRS): QMAKE_DEFAULT_LIBDIRS = /lib /usr/lib
    }
```

Which would have worked until https://github.com/Homebrew/brew/commit/79cb329f5979f5ee4b9a80c585721684b7730eaa